demo.py

```
@@ -0,0 +1,189 @@
+        except signing.SignatureError:
+            print("  tampered verify ............ REJECTED before deserialization  [attack stopped]")
+
+
+def show_merkle(records):
+    section("SEALED ARTIFACT  -- a non-degenerate Merkle tree over real weights")
+    gpt = next((r for r in records if r["model"] == "gpt10m"), None)
+    if gpt:
+        mb = gpt["artifact_size_bytes"] / 1e6
+        print(f"  gpt10m checkpoint: {gpt['num_params']:,} params -> {mb:.1f} MB safetensors")
+        print(f"  Merkle leaves (1 MB chunks): {gpt['merkle_chunk_count']}   root: {gpt['merkle_root'][:24]}...")
+        print(f"  (The original 16-char toy was 21 KB -> 1 chunk: the tree was a single hash.")
```

Ruff (0.15.17)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)